core/trino-main/src/main/java/io/trino/server/ServerMainModule.java

```diff
@@ -284,6 +291,11 @@ protected void setup(Binder binder)
         newExporter(binder).export(PauseMeter.class).withGeneratedName();
 
         configBinder(binder).bindConfig(MemoryManagerConfig.class);
+        configBinder(binder).bindConfigDefaults(MemoryManagerConfig.class, config -> {
```

Comment:
is that worth mentioning in the comment in config classes that defaults are overriden here?
Or maybe it would be more readable to define fteDefaultOverrides() in config class itself and only bind it here?